ecs_model_deployer/src/lib/schema.ts

```diff
@@ -417,6 +417,7 @@ const MetricConfigSchema = z.object({
 * @property {MetricConfig} metricConfig - Metric configuration for auto scaling.
 */
 const AutoScalingConfigSchema = z.object({
+    blockDeviceVolumeSize: z.number().min(1).default(30),
```

Comment:
I know there isn't a "good" solution to this but min 1 seems like a bad idea to allow doesn't it? Considering this field can't be updated later I'd think it would be better to keep the min as the default (30) and only allow larger values. Just a thought, but not blocking if you disagree so I'm still approving.

Reply:
Agreed, thanks for calling this out. I found that the value has to be >= 30 when doing some testing.. Need to update